src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1768,7 +1824,8 @@
         "frames": {
           "description": "An array of stack frames that represents a sequence of calls, rendered in reverse chronological order, that comprise the call stack.",
           "type": "array",
-          "minItems": 1,
+          "minItems": 0,
+          "uniqueItems": false,
```

Comment:
>uniqueItems [](start = 11, length = 11)

default on this one s/be the empty array

Reply:
There is no default because `frames` is required.

---
In reply to: [245126076](https://github.com/Microsoft/sarif-sdk/pull/1201#discussion_r245126076) [](ancestors = 245126076)